packages/extension-browser/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "jsx": "react",
         "outDir": "dist",
-        "strict": true
+        "strict": true,
+        "resolveJsonModule": true
```

Comment:
@antross since I needed the ```package.json``` file it was needed because otherwise, it was not able to read the ```package.json``` file. But now if I pull it from ```manifest.json``` I think this would not be needed.

Reply:
It's really about which kind of import/require is being used. A raw require of JSON will work without this flag, but won't have any type information (which is fine for our limited use here). This flag allows an import to be used and types dynamically generated based on the contents of the JSON file.